### PR TITLE
feat(guides): add screenshots from circleci and descriptions

### PR DESCRIPTION
**Pull request by lewisdaly**
Adds an updated description of how our ci/cd pipelines work, what we scan and check for, and what a release artifact looks like.

closes mojaloop/project#2481

### CODE REVIEW

**Comment by lewisdaly** [comment]
Images aren't loading nicely in the PR preview - you can see them all here: https://github.com/vessels-tech/documentation/blob/feat/2481-cd-cd-guide/contributors-guide/tools-and-technologies/ci_cd_pipelines.md

**Comment by elnyry-sam-k** [comment]
Thanks @lewisdaly , this looks great!

Minor change - can you also fit in the guidance to use forking WF in the PR section, maybe just a link to existing wording.. Thanks again!

**Comment by lewisdaly** [comment]
@elnyry-sam-k  you mean the workflow for how we fork repos, and submit PRs?

File: contributors-guide/tools-and-technologies/ci_cd_pipelines.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+Additionally, we also maintain a set of [Mojaloop Helm Charts](http://docs.mojaloop.io/helm/), which are 
+built from the [mojaloop/helm](https://github.com/mojaloop/helm)
+
+## Libraries
+
+> For a good example of this CI/CD pattern see [central-services-shared](https://github.com/mojaloop/central-services-shared/blob/master/.circleci/config.yml)
+
+### Pull Request (PR) Flow:
+
+The PR flow runs on pull requests, and during the PR review process, these checks must be satisfied
```
Comment (elnyry-sam-k) [inline contributors-guide/tools-and-technologies/ci_cd_pipelines.md:20]:
```suggestion
The PR flow runs on pull requests, and during the [PR review process](https://github.com/mojaloop/documentation/blob/master/contributors-guide/standards/creating-new-features.md#creating-new-features), these checks must be satisfied
```

**Review by elnyry-sam-k — APPROVED** [review]

**Pull request merged**